app/src/main/java/cz/muni/fi/pv256/movio2/uco_422476/MainFragment.java

```diff
@@ -31,7 +33,7 @@
     private OnFilmSelectListener mListener;
     private Context mContext;
     private RecyclerView mRecyclerView;
-    private ArrayList<Film> mMovieList;
+    private ArrayList<Object> mDataList;
```

Comment:
Zoznam Objectov nie je ten najlepsi napad. Lepsie by bolo definovat si interface alebo abstraktnu triedu pre prvky ktore sa mozu objavit v takomto zozname. Musia to byt predsa prvky, ktore su typu kategoria alebo film a maju bud obrazok alebo text